granian/server.py

```diff
@@ -95,6 +96,11 @@ def __init__(
         respawn_failed_workers: bool = False,
         respawn_interval: float = 3.5,
         reload: bool = False,
+        reload_paths: Sequence[pathlib.Path] = (pathlib.Path.cwd(),),
+        reload_ignore_dirs: Optional[Sequence[str]] = (),
+        reload_ignore_entity_patterns: Optional[Sequence[str]] = (),
+        reload_ignore_paths: Optional[Sequence[pathlib.Path]] = (),
+        reload_filter: Optional[BaseFilter] = None,
```

Comment:
Actually we could (or should) limit this to objs of subclasses only 🤔

The docstring of `BaseFilter` states
> Useful base class for creating filters. `BaseFilter` should be inherited and configured, rather than used directly.

so it might make more sense to use
```suggestion
        reload_filter: Optional[Type[BaseFilter]] = None,
```

Reply:
Makes sense 👍